### ISSUE TITLE
Drawer (3/21)

### ISSUE THREAD

**Issue by roblevintennis** [AgnosticUI/agnosticui#79]
leverage a11y-dialog for this -- look into teasing out the teleport portals code implementations into shareable. Toast should be doable by simply styling and positioning.

- [x] CSS
- [x] Vue 3
- [x] Vue Examples
- [x] Svelte
- [x] Svelte Examples
- [x] React
- [x] React Examples
- [x] Publish frameworks
- [x] Site Docs
